ci-operator/config/redhat-et/microshift/redhat-et-microshift-main.yaml

```diff
@@ -42,8 +41,6 @@ images:
         source_path: /go/src/github.com/redhat-et/microshift/packaging/systemd/microshift-containerized
       - destination_dir: .
         source_path: /go/src/github.com/redhat-et/microshift/packaging/systemd/microshift-containerized.service
-      - destination_dir: .
-        source_path: /go/src/github.com/redhat-et/microshift/packaging/images/microshift-aio/crio-bridge.conf
```

Comment:
we should instead copy packaging/crio.conf.d/microshift.conf to /etc/crio/crio.conf.d

Reply:
agreed, that file was merged yesterday to wait for network creation.